README.md

```diff
@@ -21,7 +21,7 @@ recommend using the [app skeleton](https://github.com/cakephp/app) as
 a starting point. For existing applications you can run the following:
 
 ``` bash
-$ composer require cakephp/cakephp:"~3.2"
+$ composer require cakephp/cakephp:"~3.3"
```

Comment:
Shouldn't the change then for releases happen for all the libraries in our `composer.json` files. Especially the behaviour for alpha releases is different.